src/app/explore/explore.component.html

```diff
@@ -67,11 +67,12 @@
           (scrolled)="onScroll()"
           [scrollWindow]="true"
           >
-          @if (this.metaCampaign?.isRegularGiving) {
-            <!-- We assume that we show regular giving campaigns only iff we are looking at a regular giving
-            metacampaign page -->
             @for (campaign of individualCampaigns; track campaign.id) {
               <div>
+                @if (campaign.isRegularGiving) {
```

Comment:
I think I'm likely to extract everything inside this if soon to go into a new Angular component to re-use in `charity.component.html`